pyproj/_crs.pyx

```diff
@@ -417,12 +425,17 @@ cdef class Base:
             The version of the WKT output.
         pretty: bool, default=False
             If True, it will set the output to be a multiline string.
+        output_axis_rule: bool, optional, default=None
+            If True, it will set the axis rule on any case. If false, never.
+            None for AUTO, that depends on the CRS and version.
+            .. versionadded:: 3.5.1
```

Comment:
Instead, I recommend adding this above parameters:

```
        .. versionadded:: 3.6.0 output_axis_rule
```

```suggestion
            .. versionadded:: 3.5.1
```

Reply:
done

Reply:
Looks great. Mind also doing that in crs.py?

Reply:
:man_facepalming: yes, of course.